### PR TITLE
Set code languages for various files (part 4)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR sets the code language for various codeblocks that don't have one set, which will help us enforce setting a language on all codeblocks. This is the fourth part.

### CODE REVIEW

File: files/en-us/web/text_fragments/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If the ID is changed or removed, the document fragment is ignored, and the link
 
 In a similar manner to document fragments, text fragments are appended onto a URL after a hash symbol (`#`). The syntax however is a bit different:
 
-```
+```url
```
Comment (hamishwillee) [inline files/en-us/web/text_fragments/index.md:38]: Didn't know we supported `url` type. Should it be used for this kind of URL, which is actually a kind of "syntax block"?

Comment (queengooborg) [reply files/en-us/web/text_fragments/index.md:38]: I don't see why not; it's technically not an invalid URL, and I don't imagine we'd ever try to lint these blocks!

File: files/en-us/web/xpath/functions/choose/index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The `choose` function returns one of the specified objects based on a boolean pa
 
 ### Syntax
 
-```
+```plain
```
Comment (hamishwillee) [inline files/en-us/web/xpath/functions/choose/index.md:15]: A bunch of these are XPath. Do we need an xpath formatter?

Comment (queengooborg) [reply files/en-us/web/xpath/functions/choose/index.md:15]: Prism doesn't support XPath and I can't find a plugin for it, so I opted to set these as plain text.

Comment (hamishwillee) [reply files/en-us/web/xpath/functions/choose/index.md:15]: I want my money back.

Comment (queengooborg) [reply files/en-us/web/xpath/functions/choose/index.md:15]: 😆

**Review by hamishwillee — APPROVED** [review]
Looks great.

I do prefer the use of plain rather than empty, because it indicates you've thought about the type. Applying this automatically would not be something I'd inflict on end users.

**Pull request merged**